china/2020_EKS_Launch_Workshop/步骤4-部署微服务以及配置ALBIngressController.md

```diff
@@ -106,9 +106,9 @@ eksctl utils associate-iam-oidc-provider --cluster=${CLUSTER_NAME} --approve --r
  * 请注意官方的policy里面包含了WAF等服务，中国区没有所以需要手动删除,修改好的已经放在resource/alb-ingress-controller目录下
 
 ```bash
-cd resource/alb-ingress-controller
+cd china/2020_EKS_Launch_Workshop/resource/alb-ingress-controller
```

Comment:
I prefer all step guide lab context is china/china/2020_EKS_Launch_Workshop,
so, we only use kubectl apply -f resource/<lab step>/xxx.yaml / or <lab step>/xxxx.yaml.